<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CutLocusPanel</class>
 <widget class="QWidget" name="CutLocusPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>19</y>
     <width>331</width>
     <height>109</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>20</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Method</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="indent">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="methodComboBox">
        <item>
         <property name="text">
          <string>Graph Distance</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Geodesics</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>70</number>
      </property>
      <item>
       <widget class="QPushButton" name="minmaxPushButton">
        <property name="text">
         <string>Min Max Points</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cutPushButton">
        <property name="text">
         <string>Cut</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
